core/src/main/java/discord4j/core/shard/GatewayBootstrap.java

```diff
@@ -194,6 +198,19 @@
         return this;
     }
 
+    /**
+     * Set the intents for the identify.
```

Comment:
I think these docs should be expanded a bit. The docs in this class are pretty extensive